packages/core/src/demo/receiver.js

```diff
@@ -15,8 +15,7 @@ const APIClient = require('@wireapp/api-client');
 const fs = require('fs');
 const {promisify} = require('util');
 const {Config} = require('@wireapp/api-client/dist/commonjs/Config');
-const {MemoryEngine} = require('@wireapp/store-engine/dist/commonjs/engine');
-const engine = new MemoryEngine();
+const {FileEngine} = require('@wireapp/store-engine');
```

Comment:
Why switching to `FileEngine`?

Reply:
Because after testing the receiver 10 times while developing I created 10 new clients :D